controls/def.cf

```diff
@@ -349,7 +349,8 @@ bundle common inventory_control
 # are disabled.
 {
   vars:
-      "lldpctl_exec" string => "/usr/bin/lldpctl";
```

Comment:
On Wed, Apr 13, 2016 at 01:30:49PM -0700, Ted Zlatanov wrote:
> @BasementTrix I added a case for `/usr/local/bin/lldpctl` as follows:
> 
> * if `/usr/sbin/lldpctl` exists, use it
> * else if `/usr/local/bin/lldpctl` exists, use it
> * else use `/usr/sbin/lldpctl` as a last resort
> 
> Try it out and see if it does the right thing for you. Thanks.

While I'm sure it will work, why not just add to the 'vars:' section
starting at line 165 which already has class stanzas for 'freebsd::'
and '!freebsd::' where the path for dmidecode is defined?


-- 
John D. "Trix" Farrar            __\\|//__               Basement.NET
trix@basement.net                (` o-o ')   http://www.basement.net/
--------------------------------ooO-(_)-Ooo--------------------------
 GPG Key Fprint: 525F DBA7 1A62 E4C4 E642  DF95 384B B851 3CEF C10A